sqlitecluster/SQLite.cpp

```diff
@@ -589,8 +588,21 @@ int SQLite::_authorize(int actionCode, const char* table, const char* column) {
         case SQLITE_FUNCTION:
             return SQLITE_OK;
             break;
-
-        // The following is the only special case where the whitelist actually applies.
+        case SQLITE_PRAGMA:
+        {
+            string normalizedTable = SToLower(table);
+            // We allow this particularly because we call it ourselves in `write`, and so if it's not allowed, all
```

Comment:
I'm confused as to what prompted this change... because it's working today, right?

Reply:
It does not. If you turn on the sqlite whitelist, then write queries are currently broken. We do not have it turned on anywhere, so nobody noticed.